core/include/scipp/core/element/util.h

```diff
@@ -4,17 +4,34 @@
 /// @author Simon Heybrock
 #pragma once
 
+#include <cstddef>
+
 #include "scipp/common/numeric.h"
 #include "scipp/common/overloaded.h"
 #include "scipp/common/span.h"
 #include "scipp/core/element/arg_list.h"
 #include "scipp/core/subbin_sizes.h"
+#include "scipp/core/time_point.h"
 #include "scipp/core/transform_common.h"
 #include "scipp/core/value_and_variance.h"
 #include "scipp/units/except.h"
 #include "scipp/units/unit.h"
 
-#include <stddef.h>
+namespace scipp::numeric {
+template <> inline bool is_linspace(const span<const core::time_point> &range) {
+  if (scipp::size(range) < 2)
+    return false;
+  if (range.back() <= range.front())
+    return false;
+
+  const auto delta = (range.back() - range.front()) / (scipp::size(range) - 1);
```

Comment:
This makes sense for floats, but maybe here we should just use the difference between the first two elements?

Reply:
Good idea